torchvision/transforms/functional.py

```diff
@@ -1528,7 +1528,7 @@ def elastic_transform(
 
         t_img = to_tensor(img)
     if not isinstance(displacement, torch.Tensor):
-        displacement = torch.from_numpy(displacement)
+        displacement = torch.Tensor(displacement)
```

Comment:
@JLrumberger this change breaks the CI and we shouldn't use `torch.Tensor` to create tensors.
Why you replaced `torch.from_numpy` ?

Reply:
@vfdev-5 I thought that `torch.from_numpy`  breaks CI and therefore I replaced it by `torch.Tensor` but I was wrong. Sorry.